interactive-player/Interactive Viewer/Managers/VideoTracksManager.swift

```diff
@@ -56,9 +56,8 @@ final actor VideoTracksManager {
 
     func observeLayerUpdates(for source: StreamSource) {
         Task { [weak self] in
-            guard let self, await self.layerEventsObservationDictionary[source.sourceId] == nil else {
-                return
-            }
+            guard let self,
+                  await self.layerEventsObservationDictionary[source.sourceId] == nil else { return }
```

Comment:
Let's follow one of these for indentation?  https://github.com/kodecocodes/swift-style-guide?tab=readme-ov-file#control-flow or https://google.github.io/swift/#control-flow-statements ? 

If it's too long for one line, like below from the kodeco swift styling guide is better ?

```
guard 
  let number1 = number1,
  let number2 = number2,
  let number3 = number3 
else {
  fatalError("impossible")
}
```

Reply:
I think that is just a personal preference. This is how I would typically format that:
```suggestion
                  guard  let number1 = number1,
                  let number2 = number2,
                   let number3 = number3  else {
                           fatalError("impossible")
                   }
```

Reply:
What I am proposing is let's follow a standard industry guideline. https://google.github.io/swift/#control-flow-statements or https://github.com/kodecocodes/swift-style-guide?tab=readme-ov-file#control-flow which are more or less the same and most accepted guideline among swift developers.

For example, the formatting changes applied file is completely unrelated to the scope of JIRA you are working on. When you are introducing new style/formatting practices and applying it to unrelated files. Examples I am referring to are `self.` to access all properties and fixing indentation to something others are not familiar with needs a broader discussion in the team and agreement.

Reply:
@sheiladoherty-dolby let's stick to the conventions which was already being followed in this repo, the team here is familiar with it and follows it as well. We can have a discussion about it in our next meeting for improvements.